### ISSUE TITLE
Reusable Information Content

### ISSUE THREAD

**Issue by truongt8** [massgov/feature-requests#268]
**Is your feature request related to a problem? Please describe.**
Feature request to allow for reusable consistent block of information across selected pages.

**Describe the solution you'd like**
What would you like to see added or changed? Why? Include any benefits, challenges, or considerations. This can be short and sweet.
Ideally, a customizable rich-text content type that we could use that we could add as an item on our pages and allow us to maintain the information in one place and automatically apply across all pages? I'm thinking something similar to the "Contact information" content type?

**Additional context**
Add any other context, use cases, or screenshots about the feature request here.

**Reference**
Include JIRA ticket number or ServiceNow ticket numbers that correspond to this issue.
ServiceNow ticket RITM0294201

**Comment by joeg8612** [comment]
Hi, can you provide a detailed description and example of the content that you would want to repeat and what pages you would put it on? We are trying to better understand the use case.

**Comment by truongt8** [comment]
Example text below that we could apply across selected pages:
We are all impacted by the social and structural policies and systems we live in, and despite the inequities that women and girls, people with disabilities, low income, BIPOC, LGBTQIA+, and immigrant individuals and families face, they are resilient in the face of adversity. Individuals and families are experts in their own lived experiences. Dismissing someone’s lived experiences causes harm. The data presented in this dashboard only shows us a part of these experiences, while people’s lives tell the entire story.

**It is important to keep all of this in mind when interpreting the data.**

Oppression impacts the [social determinants of health (SDoH),](https://www.mass.gov/guides/social-determinants-of-health) which are things that help individuals and families thrive: high-quality education, socioeconomic status, safe and affordable housing, among others.

Having the option to add an image would be a plus.

**Comment by truongt8** [comment]
Hello,
What is the average turnaround time from feature request to implementation?

**Comment by kelkins231** [comment]
I agree with this feature request - it would save a LOT of time when we require the same content across 40+ pages (which is the case for my program). It would also help wth constantly changing versions - where we would only have to update one time and it would apply across all pages. Thank you for considering!
